doc/source/security/data-at-rest-encryption.rst

```diff
@@ -142,6 +144,8 @@ The `keyring_vault_config` file has the following information:
 
 * ``secret_mount_point`` - where the `keyring_vault` stores the keys
 
+* ``secret_mount_point_version`` - the ``keyring_version`` used. Implemented in |Percona Server| 5.7.33-36. 
```

Comment:
`keyring_version` here is a bit misleading here.
The correct definition here is `KV Secrets Engine version` (`kv` or `kv-v2`) used for Vault Server's path specified by `secret_mount_point`

Reply:
Updated.